lib/logeater/event.rb

```diff
@@ -0,0 +1,11 @@
+require "active_record"
+
+module Logeater
+  class Event < ActiveRecord::Base
+    self.table_name = "events"
+
+    def self.since(timestamp)
+      self
+    end
```

Comment:
😆 What's this method for?

It's unused.

Or did you mean to extract `.where(events_table[:received_at].gteq(timestamp))` to it?

Reply:
Yeah... 😳 
I meant to extract that...